### PR TITLE
#4026 Drop JDK 8 support and migrate to JDK 11

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4026)
local test
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
QA理論上會失敗，因為我們還沒更新deps裡面裝載的JDK

File: build.gradle [hunk 1]
```diff
@@ -130,8 +130,8 @@ subprojects {
     if (isJavaModule(it.name)) {
       apply plugin: 'java'
       java {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_11
```
Comment (chia7712) [inline build.gradle:133]: @oharastream/backend 經過這隻PR後，開發後端的人都需要更新至JDK11

**Comment by chia7712** [comment]
@eechih 這隻PR會等你正式release 0.10.0後才會進行後續的處理，因為要更新deps中的JDK版本

File: build.gradle (hunk 1 above)
Comment (jackyoh) [reply build.gradle:133]: ok, 收到。

Comment (vitojeng) [reply build.gradle:133]: @chia7712 前端是否可能也要? 如果他們需要直接用 source 去跑 configurator 的話... ^^

Comment (chia7712) [reply build.gradle:133]: 沒錯XD

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry streamIT

**Pull request merged**